public/app/plugins/datasource/mysql/configuration/ConfigurationEditor.tsx

```diff
@@ -36,44 +36,44 @@ export const ConfigurationEditor = (props: DataSourcePluginOptionsEditorProps<My
     };
   };
 
-  const mediumWidth = 20;
-  const shortWidth = 15;
-  const longWidth = 40;
+  const WIDTH_SHORT = 15;
```

Comment:
This certainly feels related, but outside of the initial scope of the PR? But I also think it's fine that they're together. Thoughts?

Reply:
Figured I'd do the cleanup here since the medium width had to be updated to fit the labels in any case. Doesn't seem like that would justify and additional PR just for a minor refactor so I added it here 😄